ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -571,6 +1034,51 @@ const triggerSet: TriggerSet<Data> = {
       netRegexJa: NetRegexes.startsUsing({ id: '6A2C', source: 'ヘスペロス', capture: false }),
       response: Responses.bigAoe(),
     },
+    {
+      id: 'P4S Act Three Bait Order',
+      type: 'Tether',
+      netRegex: NetRegexes.tether({ id: '00AD', source: 'Hesperos' }),
+      netRegexDe: NetRegexes.tether({ id: '00AD', source: 'Hesperos' }),
+      netRegexFr: NetRegexes.tether({ id: '00AD', source: 'Hespéros' }),
+      netRegexJa: NetRegexes.tether({ id: '00AD', source: 'ヘスペロス' }),
+      condition: (data) => data.act === '3',
+      // Tethers come out East or West (0 seconds), (3s) Middle knockack, (6) Opposite Cardinal
+      suppressSeconds: 7,
+      infoText: (data, matches, output) => {
+        const thorn = (data.thornIds ??= []).indexOf(parseInt(matches.sourceId, 16));
+        const thornMap: { [thorn: number]: string } = {
```

Comment:
I'm not sure you need both thornMap and thornMapDirs.  I think you could call `output[thornMapDirs[thorn]]!()` and use `thornMapDirs` to handle both.

Reply:
Good catch. I was able to do so.